lib/rototiller/task/rototiller_task.rb

```diff
@@ -56,27 +59,30 @@ def add_env(*args,&block)
         add_param(@env_vars, EnvVar, attributes, args, {:set_env => true}, &block)
       end
 
-      # adds command line flags to be used in a command
-      # @param [Hash] *args hashes of information about the command line flag
-      # @option args [String] :name The command line flag
-      # @option args [String] :value The value for the command line flag
-      # @option args [String] :message A message describing the use of this command line flag
-      # @option args [String] :override_env An environment variable used to override the flag value
+      # adds command line options to be used in a command
+      # @param       [Hash]    *args hashes of information about the command line option
+      # @option args [String]  :name The command line option
+      # @option args [String]  :value The value for the command line option
+      # @option args [String]  :message A message describing the use of this command line option
+      # @option args [String]  :override_env An environment variable used to override the option value
       # @option args [Boolean] :required Indicates whether an error should be raised
-      # if the value is nil or empty string, vs not including the flag.
+      # if the value is nil or empty string, vs not including the option.
       #
       # for block {|a| ... }
-      # @yield [a] Optional block syntax allows you to specify information about the command line flag, available methods track hash keys
-      def add_flag(*args, &block)
-        raise ArgumentError.new("add_flag takes a block or a hash") if !args.empty? && block_given?
+      # @yield [a] Optional block syntax allows you to specify information about the command line option,
+      #            available methods track hash keys
+      def add_option(*args, &block)
+        raise ArgumentError.new("add_option takes a block or a hash") if !args.empty? && block_given?
         attributes = [:name, :default, :message, :override_env, :required]
         add_param(@flags, CommandFlag, attributes, args, &block)
       end
+      alias_and_deprecate :add_flag, :add_option
```

Comment:
Once the API changes will we have to deprecate both these methods without aliasing them?